core/exchangeChecker.js

```diff
@@ -12,10 +14,15 @@ Checker.prototype.notValid = function(conf) {
     return this.cantMonitor(conf);
 }
 
+Checker.prototype.getExchangeCapabilities = function(slug) {
+  var Trader = require(dirs.exchanges + config.watch.exchange);
+  return Trader.getCapabilities();
+}
+
 // check if the exchange is configured correctly for monitoring
 Checker.prototype.cantMonitor = function(conf) {
   var slug = conf.exchange.toLowerCase();
-  var exchange = _.find(exchanges, function(e) { return e.slug === slug });
+  var exchange = this.getExchangeCapabilities(slug);
 
   if(!exchange)
```

Comment:
this if check won't get hit anymore, because [this line](https://github.com/askmike/gekko/pull/750/commits/02a62896e4d27adc2e04bc29f1382da9680461c7#diff-2a878ed84810bccfebe021d0e89d7ca8R18) will crash if the file is not found.